index.js

```diff
@@ -68,3 +69,16 @@ function sortByLast(a, b) {
   }
   return 0
 }
+
+function search() {
+  $("#search").on("keyup", function(event) {
```

Comment:
A couple things:

I see you're adding `event` as an argument here, but not calling it in the function. If you wanted to use `event.target.id` as an identifier or `event.target.value` to get the value you could, but you don't have to. If you're not going to use it in the function, I would remove it from the arguments.

You're using `$("#search")` a couple times in this function. Each time jQuery is scanning the markup for that target every time you do that. It's better style to save that as a variable: e.g. `const $search = $("#search");`